guide/src/installation.md

```diff
@@ -11,7 +11,7 @@ There are multiple ways to install Yarner:
 2. Unzip somewhere
 3. *Optional:* add directory `yarner` to your `PATH` environmental variable
 
-## From Git using `cargo`
+## From GitHub using `cargo`
```

Comment:
I really think we should publish to crates.io to simplify things for potential users.

Reply:
Now that you started to clean up the code, sure! 😉

Reply:
So, this should go into a CI MR as you we should probably publish tags automatically via CI...